tests/mock_tables/state_db.json

```diff
@@ -24,6 +24,13 @@
     "manufacturer": "VENDOR_NAME",
     "model": "MODEL_NAME"
   },
+  "TRANSCEIVER_INFO|Ethernet4": {
+    "type": "QSFP+",
+    "hardware_rev": "\xff\xff",
```

Comment:
>\xff\xff [](http://example.com/codeflow?start=21&length=8)

Let's add another test case `\xdcff\xdcff` to trigger another bug we found during debugging.

Reply:
Raised a github issue , where the bad string can cause agentx crash in 2021/master branch.
https://github.com/Azure/sonic-snmpagent/issues/239.
This issue will be seen on branches >= 2020x .